templates/sso73-image-stream.json

```diff
@@ -18,11 +18,11 @@
                     "description": "Red Hat Single Sign-On 7.3 continuous delivery",
                     "openshift.io/display-name": "Red Hat Single Sign-On 7.3 continuous delivery",
```

Comment:
And also replace `registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift` (two times below, one for `:latest` and one for `:1.0`) with `registry.redhat.io/redhat-sso-7/sso73-openshift`(and add `:latest` and `:1.0` tags to respective places).